src/disco/components/Addon/styles.scss

```diff
@@ -96,6 +96,10 @@ $addon-padding: 20px;
       padding: 0 15px;
     }
 
+    @include respond-to(extraSmall) {
+      display: none;
+    }
+
```

Comment:
How about this instead? (since we are mobile-first?)

```diff
diff --git a/src/disco/components/Addon/styles.scss b/src/disco/components/Addon/styles.scss
index 522521424..7abab98de 100644
--- a/src/disco/components/Addon/styles.scss
+++ b/src/disco/components/Addon/styles.scss
@@ -85,10 +85,11 @@ $addon-padding: 20px;
   .logo {
     align-items: center;
     align-self: stretch;
-    display: flex;
+    display: none;
     padding: 0 5px;
 
     @include respond-to(medium) {
+      display: flex;
       padding: 0 10px;
     }
 
@@ -96,10 +97,6 @@ $addon-padding: 20px;
       padding: 0 15px;
     }
 
-    @include respond-to(extraSmall) {
-      display: none;
-    }
-
     img {
       display: block;
       height: 64px;
```